### PR TITLE
docs: add small usage

### PR DESCRIPTION
**Pull request by david-caro**
* Adds a small usage 'howto' to prevent the user from having to jump
  to even another docs page in order to get started with the template
  usage

Signed-off-by: David Caro <david@dcaro.es>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -70,3 +70,41 @@ To generate correct files, please provide the following input to Cookiecutter:
...
+--------------
+This example might be deprecated at the time you read it, but it will give you
+a good idea on how to use this template to generate a new project.
+
+Requirements:
+==============
+You will need to have installed the Cookiecutter tool, we recommend to use a
+virtualenv, specially with virtualenvwrapper to help you around. So assuming
+that you have virtualenvwrapper already installed (remember to add `source
+virtualenvwrapper.sh` to your bashrc and relod the console or source it
```
Comment (lnielsen) [inline README.rst]: ``/relod/reload/``

File: README.rst [hunk 2]
```diff
@@ -70,3 +70,41 @@ To generate correct files, please provide the following input to Cookiecutter:
...
+directly after installing it), you can just run::
+
+    $ mkvirtualenv cookiecutter
```
Comment (lnielsen) [inline README.rst]: Perhaps use ``.. code-block:: console``?

Comment (david-caro) [inline README.rst]: Is it necessary? the `::` option is essentially the same (without syntax highlight, though I don't think it helps here) and way easier to read and write on the text file.

**Comment by david-caro** [comment]
hmmm... I don't like how github allows you to edit files online... I'll squash the new commit. Sorry for the noise.

File: README.rst [hunk 3]
```diff
@@ -70,3 +70,41 @@ To generate correct files, please provide the following input to Cookiecutter:
                                  super project of this module, e.g. Invenio
                                  itself.
 ================================ =============================================
+
+Usage example
+--------------
```
Comment (jirikuncar) [inline README.rst]: Can you keep the underlining same length as the heading?

File: README.rst [hunk 4]
```diff
@@ -70,3 +70,41 @@ To generate correct files, please provide the following input to Cookiecutter:
...
+This example might be deprecated at the time you read it, but it will give you
+a good idea on how to use this template to generate a new project.
+
+Requirements:
```
Comment (jirikuncar) [inline README.rst]: Do we need the colon?

Comment (david-caro) [inline README.rst]: Not really, I'll remove

File: README.rst (hunk 3 above)
Comment (david-caro) [inline README.rst]: I was creating a section called 'Usage example' at the same level as 'configuration', with the others as subsections (requirements, create a new python module...).

It makes no sense to me to put them all at the same level.

Comment (david-caro) [inline README.rst]: Oh, you mean as in 'the number of chars in the underlining the exact same number as the header'?

File: README.rst [hunk 5]
```diff
@@ -70,3 +70,41 @@ To generate correct files, please provide the following input to Cookiecutter:
...
+Requirements
+============
+You will need to have installed the Cookiecutter tool, we recommend to use a
+virtualenv, specially with virtualenvwrapper to help you around. So assuming
+that you have virtualenvwrapper already installed (remember to add `source
+virtualenvwrapper.sh` to your bashrc and reload the console or source it
+directly after installing it), you can just run::
+
+    $ mkvirtualenv cookiecutter
+    $ pip install --upgrad pip setuptools  # to make sure we have the latest
```
Comment (jirikuncar) [inline README.rst]: `--upgrade`

File: README.rst [hunk 6]
```diff
@@ -70,3 +70,41 @@ To generate correct files, please provide the following input to Cookiecutter:
                                  super project of this module, e.g. Invenio
                                  itself.
 ================================ =============================================
+
+Usage example
+-------------
+This example might be deprecated at the time you read it, but it will give you
```
Comment (jirikuncar) [inline README.rst]: > This example might be deprecated at the time you read it ...

It doesn't give a good impression to readers.

Comment (david-caro) [inline README.rst]: It's way better than finding out it's deprecated after having to debug it

**Comment by jirikuncar** [comment]
- [ ] `docs: addition of package usage`
- [ ] `git rebase upstream/master`

**Comment by jirikuncar** [comment]
Closing for inactivity. @david-caro please re-open when you fix typo and rebase.

**Pull request closed without merging**